tap-agent/src/agent/sender_allocation.rs

```diff
@@ -132,6 +198,13 @@ impl Actor for SenderAllocation {
             );
         })?;
 
+        //Since this is only triggered after allocation is closed will be counted here
+        CLOSED_ALLOCATIONS.inc();
+
+        RAVS_PER_SENDER_ALLOCATION
+            .with_label_values(&[&sender_allocation])
+            .inc();
```

Comment:
You probably should set this at the end of `rav_requester_single()` right before the Ok(())